packages/launch-config/test/launch-config-crud/update.test.ts

```diff
@@ -34,7 +34,7 @@ describe('update', () => {
     test('Create and then update existing launch config in launch.json', async (): Promise<void> => {
         // create a new
         const launchJSONPath = join(TestPaths.feProjectsLaunchConfig);
-        let result = await createLaunchConfig(
+        let result: any = await createLaunchConfig(
```

Comment:
Why not using the real type `Editor`?
```suggestion
        let result: Editor = await createLaunchConfig(
```